Team6220_2021/src/main/java/org/firstinspires/ftc/team6220_2021/TeleOpComp.java

```diff
@@ -19,10 +18,12 @@
     DcMotor motorFrontLeft;
```

Comment:
You should not define your motors here.  They should already be defined in MasterOpMode.  Don't define them more than once.  Don't reverse motors here.  Reverse them once, and stick to the same convention throughout all programs.